components/Footer.js

```diff
@@ -0,0 +1,66 @@
+import Link from "next/link";
+import styled from "styled-components";
+import { useRouter } from "next/router";
+import { TbBook2 } from "react-icons/tb";
+import { IconContext } from "react-icons";
+
+function Footer() {
+  const router = useRouter();
+  
+  return (
+    <footer>
+      <NavBar>
+        <Link href="/tutorials" passHref>
+          <NavItem isActive={router.pathname === "/tutorials"}>
```

Comment:
Just a small suggestion: You could deconstruct the pathname from useRouter then you don't need to write router.pathname very time ;)

Reply:
Very good idea! Thank you, @CathrinTruchan !